chainerio/__init__.py

```diff
@@ -84,44 +184,98 @@ def create_handler(uri: str) -> IO:
     return handler
 
 
-def isdir(file_path: str) -> bool:
+def isdir(path: str) -> bool:
+    """return whether the given path is a directory
+
+    Works similar to the `os.isdir` and shares the same parameter list.
+
+    Args:
+        path (str): the path to the target directory
+
+    Returns:
+        True when the path points to a directory, False when not
+
+    """
     global _DEFAULT_CONTEXT
     default_context = _DEFAULT_CONTEXT
 
     (handler, actual_path, is_URI) = \
-        default_context.get_handler_by_name(file_path)
+        default_context.get_handler_by_name(path)
     return handler.isdir(actual_path)
 
 
-def mkdir(file_path: str, mode: int = 0o777, **kwargs) -> None:
+def mkdir(path: str, mode: int = 0o777, **kwargs) -> None:
+    """make a directory with mode
+
+    Works similar to the `os.makedir` and shares the same parameter list.
+
+    Args:
+        path (str): the path to the directory to make
+
+        mode (int): the mode of the directory
+
+    """
     global _DEFAULT_CONTEXT
     default_context = _DEFAULT_CONTEXT
 
     (handler, actual_path, is_URI) = \
-        default_context.get_handler_by_name(file_path)
+        default_context.get_handler_by_name(path)
     return handler.mkdir(actual_path, mode, *kwargs)
 
 
-def makedirs(file_path: str, mode: int = 0o777,
+def makedirs(path: str, mode: int = 0o777,
              exist_ok: bool = False) -> None:
+    """make directories recursively with mode
```

Comment:
"Makes directories recursively with mode"

Reply:
Addressed in e0df1ba